orderapi/src/main/java/fi/hel/verkkokauppa/order/api/data/accounting/AccountingSlipRowDto.java

```diff
@@ -51,6 +52,8 @@ public class AccountingSlipRowDto {
     @JacksonXmlProperty(localName = "FunctionalArea")
     private String functionalArea;
 
+    private AccountingRowTypeEnum rowType;
```

Comment:
Should this have JsonIgnore or not?

Reply:
Its done in AccountingSlipRowMixInDto like rest of the 'extra' fields. Added comment here